From e0836121b6aa440c1a1d52c59431292d20fd7e35 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Thu, 30 Nov 2017 11:30:30 +0800
Subject: [PATCH] arm64: dts: rockchip: add cpuinfo dt node for
 rk3399-android-6.0

Add cpuinfo dt node for android-6.0 on rk3399 platform.
Sync from rk3399-android.dtsi

Change-Id: I18dd17ac1b78d59932f40a0796faafe942fa389d
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android-6.0.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android-6.0.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android-6.0.dtsi
index 657fd6c04283..5f6b21ba9ca3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android-6.0.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android-6.0.dtsi
@@ -55,6 +55,12 @@
 		bootargs = "earlycon=uart8250,mmio32,0xff1a0000 swiotlb=1";
 	};
 
+	cpuinfo {
+		compatible = "rockchip,cpuinfo";
+		nvmem-cells = <&efuse_id>;
+		nvmem-cell-names = "id";
+	};
+
 	ramoops_mem: ramoops_mem {
 		reg = <0x0 0x110000 0x0 0xf0000>;
 		reg-names = "ramoops_mem";
-- 
2.35.3

